### ISSUE TITLE
Critical error trying to view manifest

### ISSUE THREAD

**Issue by codingpandaren** [mijora/omniva-prestahop-1.7#24]
**Prestashop version:** 1.7.2.4
**Module version:** 1.1.5
**PHP version:** 7.0.33

**Issue 1:** "Lemtinga klaida" (en. critical error) is shown when trying to view manifests. 
**Issue 2:**  When trying to view label (in Shipping->Omniva užsakymai) shows a blank PDF. Same blank PDF is generated when trying to print label from order page.
**Issue 3:** In order page when "Generuoti lipduka" (en. Generate label) is clicked error "No saved barcodes received" is shown.

Also I'd like to know what PHP version is recommended and what are other important requirements for module to work correctly.

**Comment by LithMage** [comment]
Hi its fine with 7.0
Important things are - overrides must be enabled, correct module settings.

For PDF problems try to change lines in controllers/front/omnivaltadminajax.php (there is a couple of them)
```
require_once(_PS_MODULE_DIR_.'omnivaltshipping/tcpdf/tcpdf.php');
```
to 
```
require_once(_PS_TOOL_DIR_ . 'tcpdf/tcpdf.php');
```

we noticed some prestashop installations already have TCPDF library and thus conflicts with the one in module.

if problems persist please contact us at omniva@mijora.lt (in lithuanian or english) so we can further track down the issue

**Comment by codingpandaren** [comment]
Tried doing the changes, but apparently there is no TCPDF library by default in my Prestashop because this error is being thrown

`********/tools/tcpdf/tcpdf.php): failed to open stream: No such file or directory`

I contacted your provided email, thank you.

**Comment by LithMage** [comment]
Closing this issue since the problem was with incorrect module settings entered.
